### PR TITLE
Frontend/feature/timer

### PR DESCRIPTION
**Pull request by SzymonGlab**
Created timer component, ready to receive data from server.

### CODE REVIEW

File: frontend/src/components/GameView/GameView.js [hunk 1]
```diff
@@ -18,12 +20,17 @@ export default function GameView() {
     [1, 2, 3, 4, 5, 6, 7, 8, 9]
   ];
 
+  const mockTime = { max_time: 300, current_time: 6 };
```
Comment (wadamek65) [inline frontend/src/components/GameView/GameView.js]: This is not enforced, but try to keep one convention of naming. In this case I would suggest changing the snake case to camel case.

Comment (wadamek65) [reply frontend/src/components/GameView/GameView.js]: Also, doesn't make sense to put those values in an object if you are gonna pass them separately anyway. Either make the prop accept the whole object or put them in two separate values.

File: frontend/src/components/GameView/Timer/Timer.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const progress = Math.floor((currentTime / maxTime) * 100);
+  const minutes = Math.floor(currentTime / 60);
+  let seconds = currentTime - minutes * 60;
+
+  let textColor = timer.white;
+  let strokeColor = timer.base;
+  const red = "#cc0000";
+
+  if (seconds < 60) {
+    seconds = `0${seconds}`;
```
Comment (wadamek65) [inline frontend/src/components/GameView/Timer/Timer.js:16]: I think you meant to check if `second < 10` not `60`? Currently it would show seconds as `059` I believe.

File: frontend/src/components/GameView/GameView.js (hunk 1 above)
Comment (wadamek65) [reply frontend/src/components/GameView/GameView.js]: Or another solution is to name the object keys accordingly to props so: `const mockTime = { currentTime: 6, maxTime: 300 }` and then you can pass it as props like this: `<Timer {...mockTime} />`

File: frontend/src/components/GameView/Timer/Timer.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    float: right;
+    font-family: 'Exo 2', sans-serif;
+}
+
+@media(max-width: 1000px){
+    .Timer p{
+        float: none;
+        text-align: center;
+    }
+
```
Comment (wadamek65) [inline frontend/src/components/GameView/Timer/Timer.scss]: Unnecessary whitespace

File: frontend/src/components/GameView/Timer/Timer.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
```
Comment (wadamek65) [inline frontend/src/components/GameView/Timer/Timer.scss:23]: No new line

**Review by wadamek65 — APPROVED** [review]

File: frontend/src/components/GameView/GameView.js (hunk 1 above)
Comment (SzymonGlab) [reply frontend/src/components/GameView/GameView.js]: Changed as suggested.

File: frontend/src/components/GameView/Timer/Timer.scss (hunk 4 above)
Comment (SzymonGlab) [reply frontend/src/components/GameView/Timer/Timer.scss:23]: Removed

File: frontend/src/components/GameView/Timer/Timer.js (hunk 2 above)
Comment (SzymonGlab) [reply frontend/src/components/GameView/Timer/Timer.js:16]: Yes ofc, I've changed it to 10.

File: frontend/src/components/GameView/Timer/Timer.scss (hunk 3 above)
Comment (SzymonGlab) [reply frontend/src/components/GameView/Timer/Timer.scss]: Which one? @wadamek65

Comment (wadamek65) [reply frontend/src/components/GameView/Timer/Timer.scss]: new line between brackets, visually inconsistent with other formatting

Comment (SzymonGlab) [reply frontend/src/components/GameView/Timer/Timer.scss]: There was a separate comment, about new line, so I must had misunderstood.

**Review by karolszymonczyk — APPROVED** [review]
